Automattic/jetpack
# AI Assistant: introduce jetpack/ai supports

This PR introduces the `jetpack/ai` support to extend other blocks easier:

* Check if the blocks have the `jetpack/ai` supports
* Extended the `core/paragraph` and `core/header`
* Add a super simple AiAssistantPanel


Fixes #

## Proposed changes:

* AI Assistant: introduce jetpack/ai supports

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Confirm the `core/paragraph` and `core/heading` block types get extended as long as:
  * `ai-assistant-support` beta extension is enabled
  * site is connected
  * it doesn't require an upgrade 

<img width="840" alt="Screenshot 2023-06-12 at 16 44 41" src="https://github.com/Automattic/jetpack/assets/77539/9e0333de-6675-4402-852e-a4a832cfc713">
